pylint/lint/parallel.py

```diff
@@ -168,8 +169,5 @@ def check_parallel(
             all_mapreduce_data[worker_idx].append(mapreduce_data)
             linter.msg_status |= msg_status
 
```

Comment:
Why did you remove the close and join here? These were added explicitly at some point, even though the context manager should handle these normally. Is there a specific reason for removing them? Doesn't `ProccessPoolExecutor` support them?

Reply:
> Is there a specific reason for removing them? Doesn't ProccessPoolExecutor support them?

Exactly, these are not supported by `ProccessPoolExecutor`. The context manager calls [shutdown](https://docs.python.org/3/library/concurrent.futures.html#concurrent.futures.Executor.shutdown) upon exiting which is explicitly mentioned in the docs.